src/app/projects/components/projects/projects.component.ts

```diff
@@ -168,10 +170,12 @@ export class ProjectsComponent implements OnInit, OnDestroy {
       projectList.sort((a, b) => a.name.localeCompare(b.name));
       projectList.forEach(projectItem => {
         if (projectItem.createdAt) {
-          projectItem.timeStamp = parseUTC(projectItem.createdAt);
-          const splitDateTime = projectItem.timeStamp.split(',');
-          projectItem.date = splitDateTime[0].trim();
-          projectItem.time = splitDateTime[1];
+          const projectItemCopy = jsonCopy(projectItem);
+          projectItemCopy.timeStamp = parseUTC(projectItemCopy.createdAt);
+          const splitDateTime = projectItemCopy.timeStamp.split(',');
+          projectItemCopy.date = splitDateTime[0].trim();
+          projectItemCopy.time = splitDateTime[1];
+          projectItem = projectItemCopy;
```

Comment:
Why is a copy used here? What would happen, if we just use the original projectItem object and change the date and time information directly on it?

Reply:
Some object in Typescript cannot be extended (in the query we specify which fields are coming from BE) and if we want to extend the object we need a copy and then add the new object to the list.

Reply:
Ah, that makes sense, thanks for the explanation